src/parser/parser.yy

```diff
@@ -2714,6 +2723,7 @@ traverse_sentence
     | delete_edge_sentence { $$ = $1; }
     | show_queries_sentence { $$ = $1; }
     | kill_query_sentence { $$ = $1; }
+    | describe_user_sentence { $$ = $1; }
```

Comment:
The traverse_sentence means supports input/output. Don't it here.

Reply:
The desc_user_sentcence supports output pipe.

Reply:
Why need this? Normally, admin sentence don't support it.

Reply:
To Support `desc user user1 | YIELD $-.role where $-.space == "space3" `

Reply:
Does this pass in discussion meeting?

Reply:
@yixinglu From Yee‘s conversation

Reply:
@yixinglu From Yee‘s conversation